index.html

```diff
@@ -12,7 +12,7 @@
 </head>
 <body>
     <header>
-        <div class="my-header">
+        <div class="my-headers">
             <a class="logo" href="index.html">Larry Villegas</a>
```

Comment:
the logo's  contrast (4.44:1) is low
the contrast between text and background colors is very low